roles/common/tasks/main.yml

```diff
@@ -33,8 +33,22 @@
     #- fontconfig # needed for phantomjs
     #- nfs-common # make virtualbox faster
 
-- name: copy up .bashrc
-  template: src=bashrc.j2 dest=/{{ home_path }}/.bashrc
+- name: copy up .bashrc (admin user)
```

Comment:
Maybe we can just add rbenv setup to the .bashrcs instead of replacing the whole file. The file might have other important customisations.